src/main/resources/celebrateHerPage.json

```diff
@@ -0,0 +1,26 @@
+{
+  "id": "page:CELEBRATE_HER",
+  "heroSection": {
+    "title": "Celebrate Her",
+    "images": [
+      {
+        "path": "https://cloudprovider.com/image.png",
+        "alt": "There is a group of women showing WCC logo",
+        "type": "desktop"
+      }
+    ]
+  },
+  "page": {
+    "description": "Women Coding Community highlights the achievements of our members by celebrating them with Celebrate Her"
+  },
+  "data": {
+    "items": [
+      {
+        "hashtag": "#celebrate_her",
```

Comment:
I think this part is not necessary because it always be using this tag, unless it is a list and we could add more.